service/src/main/java/bio/terra/workspace/service/iam/SamService.java

```diff
@@ -130,6 +130,22 @@ public String getUserEmailFromSam(AuthenticatedUserRequest userRequest)
     }
   }
 
+  /**
+   * Gets proxy group email.
+   *
+   * <p>This takes in userEmail instead of AuthenticatedUserRequest because of
+   * WorkspaceService.removeWorkspaceRoleFromUser(). When User A removes User B from workspace, we
+   * want to get B's proxy group, not A's.
+   */
+  public String getProxyGroupEmail(String userEmail, String token) throws InterruptedException {
```

Comment:
Is this user A or B?

Reply:
A. Though I don't think we need to clarify in a comment. It's just a token the caller passed in, for calling SAM.